### PR TITLE
[ENUM] Adds missing Jug Pet to petId enum table

### PR DESCRIPTION
**Pull request by hooksta4**
**_I affirm:_**
- [x] I understand that if I do not agree to the following points by completing the checkboxes my PR will be ignored.
- [x] I understand I should leave resolving conversations to the LandSandBoat team so that reviewers won't miss what was said.
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/LandSandBoat/server/blob/base/CODE_OF_CONDUCT.md).
- [x] I have _**tested my code and the things my code has changed**_ since the last commit in the PR and will test after any later commits.

## What does this pull request do?
Adds missing jug pets to petId enum table.

### CODE REVIEW

**Review by Xaver-DaRed — DISMISSED** [review]

File: scripts/enum/pet_id.lua [hunk 1]
```diff
@@ -57,6 +57,26 @@ xi.petId =
...
+    AUDACIOUS_ANNA        = 59,
+    PRESTO_JULIO          = 60,
+    BUGEYED_BRONCHA       = 61,
+    GOOEY_GERARD          = 62,
+    GOREFANG_HOBS         = 63,
+    FAITHFUL_FALCOR       = 64,
+    CRUDE_RAPHIE          = 65,
+    DAPPER_MAC            = 66,
+    SLIPPERY_SILAS        = 67,
+    TRUBID_TOLOI          = 68,
```
Comment (almuth150) [inline scripts/enum/pet_id.lua]: Typo here - this should be `TURBID_TOLOI`.

https://www.bg-wiki.com/ffxi/Auroral_Broth

Comment (WinterSolstice8) [reply scripts/enum/pet_id.lua]: Good catch, thanks

Comment (hooksta4) [reply scripts/enum/pet_id.lua]: Thanks Almuth, you're the best.

**Comment by hooksta4** [comment]
Corrected typo - thanks.

**Review by Xaver-DaRed — APPROVED** [review]

**Review by WinterSolstice8 — APPROVED** [review]

**Pull request merged**